### ISSUE TITLE
add the RRU's status in the livemap

### ISSUE THREAD

**Issue by parmentelat** [fit-r2lab/r2lab.inria.fr#5]
from mattermost, by @turletti 

> Hello @ThierryParmentelat Is it possible to add in the R2lab map the status of RRUs jaguar, panther X310, N300 and N310 ? 

and
```
X310 : on top of the rack
N300: fit01
N320: fit03
jaguar: fit08
panther: fit11
```

**Comment by parmentelat** [comment]
@turletti 
can you point at logos that we could use for these 5 devices ?

**Comment by parmentelat** [comment]
@turletti 
also, I expect this information **replaces** the current gnuradio thingy ? or is in **in addition to** the gnuradio logo ?

**Comment by turletti** [comment]
It is in addition to current logos. Only new antennas are fixed at these places with no link to corresponding fit nodes

**Comment by turletti** [comment]
Regarding the logo, it's a 5G MIMO antenna (it looks like a plate). Either we use a "MIMO" text logo or somethink like )))

**Comment by parmentelat** [comment]
I'm confused as to where these status should go inside the livemap, because at the same time you said that

> It is in addition to current logos. Only new antennas are fixed at these places with no link to corresponding fit nodes

but in this case what did you mean with the initial list of devices, when you said for example 
`N300: fit01`

**Comment by turletti** [comment]
I think the most useful and important is to put the status of RRUs in 
the R2lab livemap at the place the antennas are located, e.g. for the 
N300, on fit01.

Indeed, it is important to check before running a new R2lab experiment 
that those RRUs are off and not transmitted...

As all the sopnode servers are assumed to run H24, there is no real need 
to list them in the same place than the 37 fit nodes.

And since the RRUs are connected through the different switches to any 
of the sopnode servers, it would be complex to check which server node 
is currently handling N300 for instance.

Thierry

Le 17/01/2023 à 11:17, parmentelat a écrit :

**Comment by parmentelat** [comment]
look, I had no intention to add the status of the sopnodes :)

what I am saying is just this: the real estate on the livemap is scarce already
and so if we're really talking about adding information, basically at the same location 
as the one where fit01 gets displayed already, we're going to need to be careful to not overload the graphics

plus, regarding x310, it's going to be basically at the same location as phone1, which is an even more populated area in the map...

**Comment by turletti** [comment]
I understand but the status of those RRUs is important for the R2lab 
platform. Perhaps using smaller icons when off and bigger when on ?

Thierry

Le 18/01/2023 à 09:18, parmentelat a écrit :

**Comment by parmentelat** [comment]
there is a temporary version of the livemap online with the phones taken out of the maps
plus visual additions on mouse hover, like so
![Image Pasted at 2023-1-22 18-53](https://user-images.githubusercontent.com/4519227/214612445-70f2f0e8-729d-4236-b6fc-924b2c201b3b.png)

I'm putting this on hold, and will now refactor rhubarbe wrt PDUs so that I can write a more decent monitoring tool for PDUs, and once that is done I will add the PDU-controlled devices in the map in a similar way as the phones

**Comment by parmentelat** [comment]
we have a first implementation live
